### PR TITLE
OSIAM 3.0

### PR DESCRIPTION
**Pull request by dacrome**

### CODE REVIEW

**Comment by dacrome** [comment]
Blocked by osiam/distribution/pull/37

File: bootstrap.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 
-OSIAM_VERSION=2.4
+OSIAM_VERSION=3.0
```
Comment (tkrille) [inline bootstrap.sh:3]: I think we want a SNAPSHOT version. It just hasn't been updated after the last release :)

Comment (tkrille) [inline bootstrap.sh:3]: My fault, we need a distribution, of course. So no snapshots.

**Comment by tkrille** [comment]
Ok, I see. These are just for preparation?

**Comment by dacrome** [comment]
> Ok, I see. These are just for preparation?

Yes

**Comment by dacrome** [comment]
I close this PR because the vagrant VM now uses the [current GitHub sources](https://github.com/osiam/vagrant/pull/23), so it can also be created with the OSIAM 3.0 release, e.g. by [setting the ref from `master` to `3.0`](https://github.com/osiam/vagrant/blob/bf896c33dffcde77e7472f97e11687c6b043e3f0/install/build.conf)

**Pull request closed without merging**